### PR TITLE
core: added `GetMouseDelta()`

### PR DESCRIPTION
**Pull request by adricoin2010**
Thanks raysan for letting me know about `Mouse.previousPosition`.

This pull request adds `GetDeltaMouse()`. It is a simple function that returns the movement of the mouse for each frame.

It can be useful for interfaces, cameras, etc.

### CODE REVIEW

File: src/core.c [hunk 1]
```diff
@@ -3540,6 +3540,17 @@ Vector2 GetMousePosition(void)
     return position;
 }
 
+// Mouse movement per frame XY
+Vector2 GetMouseDelta()
```
Comment (raysan5) [inline src/core.c:3544]: Functions in C require the input parameter definition: `GetMouseDelta(void)`, your declaration implies an indeterminate number of input parameters.

File: src/raylib.h [hunk 2]
```diff
@@ -1104,6 +1104,7 @@ RLAPI bool IsMouseButtonUp(int button);                       // Check if a mous
 RLAPI int GetMouseX(void);                                    // Get mouse position X
 RLAPI int GetMouseY(void);                                    // Get mouse position Y
 RLAPI Vector2 GetMousePosition(void);                         // Get mouse position XY
+RLAPI Vector2 GetMouseDelta();                                // Mouse movement per frame XY
```
Comment (raysan5) [inline src/raylib.h:1107]: Please, could you review the function description to be consistent with all the other related function descriptions? -> `Get mouse delta between frames`.

**Comment by raysan5** [comment]
@adricoin2010 Hi! I added some review points, did you check them?

**Comment by adricoin2010** [comment]
@raysan5 Hi, thanks for your patience, I was a bit busy. I have made the necessary changes.
I was not aware of the problems of not adding `void` as an input parameter. Everyday you learn something new.

**Comment by raysan5** [comment]
@adricoin2010 thanks for the review!

**Pull request merged**